exodus/exodus/settings/common_dev.py

```diff
@@ -3,31 +3,29 @@
 from .base import *
 
 default_secret_key = '9b80473f1b0c7d9f1859cfa754e40e26'
-SECRET_KEY = os.environ.get('EXODUS_SECRET_KEY', default_secret_key)
+SECRET_KEY = env('EXODUS_SECRET_KEY', default=default_secret_key)
 
-DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
 ALLOWED_HOSTS = ['*']
 
 STATIC_URL = '/static/'
-STATIC_ROOT = '/static/'
-STATICFILES_DIRS = (os.path.join(BASE_DIR, '..', 'static'), )
+STATIC_ROOT = f'{BASE_DIR}/static/'
+STATICFILES_DIRS = [str(APPS_DIR / "static")]
```

Comment:
```suggestion
STATICFILES_DIRS = [f'{APPS_DIR}/static']
```

Should do the same but looks more consistent and readable to me

Reply:
Agree

Reply:
Done